cmd/snap/cmd_quota.go

```diff
@@ -377,3 +400,46 @@ func processQuotaGroupsTree(quotas []*client.QuotaGroupResult, handleGroup func(
 	}
 	return processGroups(roots)
 }
+
+var MemoryCGroupFunc = getMemoryCGroupOutput
+
+func getMemoryCGroupOutput() ([]string, error) {
+	grep := exec.Command("grep", "memory")
+	catCgroups := exec.Command("cat", "/proc/cgroups")
```

Comment:
(nitpick^3) probably `if strings.HasPrefix(line, "memory ") {` (notice the extra space) to avoid matching on "memory-new-thing-that-got-added-years-later-but-has-the-same-prefix")

Reply:
switched to memory\t as the file is separated by tabs